packages/scm/src/browser/scm-tree-widget.tsx

```diff
@@ -567,7 +588,7 @@ export class ScmResourceComponent extends ScmElement<ScmResourceComponent.Props>
                 model,
                 treeNode
             }}>
-                <div title={tooltip} className='status' style={{ color }}>
+                <div title={tooltip} className={themeIconClass + ' ' + classType} style={{ color, backgroundImage }} >
```

Comment:
I think it'd be cleaner if we create a `className` array and perform `join(' ')` here instead.

Reply:
It would also prevent the html from looking like `class=''  foo"` with leading whitespaces.

Reply:
Thanks, that works better.
It is also done